src/styles/stars.scss

```
@@ -0,0 +1,26 @@
+.card__stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding-top: 16px;
+}
+
+.stars.stars--4 {
+  display: flex;
+  align-items: center;
+  height: 16px;
+  padding-left: $padding-left;
+}
+
+.stars__star {
```

```suggestion
.stars {
  &--4 {
    display: flex;
    align-items: center;
    height: 16px;
    padding-left: $padding-left;
  }

  &__ star {
}
```
again - use SCSS nesting, like here :arrow_up: